hack/force-cert-retation.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -xe
+
+KUBECTL=${KUBECTL:-kubectl}
+POD_NAMESPACE=${POD_NAMESPACE:-kubemacpool-system}
+POD_APP_LABEL=${POD_APP_LABEL:-kubemacpool-leader=true}
+WEBHOOK_TYPE=${WEBHOOK_TYPE:-mutatingWebhookConfiguration}
```

Comment:
ditto

Reply:
Let's remove this attribute and do it at all webhookconfigurations mutating or validating.